common/smoke-test/smoke-tests.yml

```diff
@@ -6,30 +6,43 @@ jobs:
           OSVmImage: "ubuntu-18.04"
           SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
           ArmTemplateParameters: $(AzureCloudArmTemplateParameters)
+          NodeTestVersion: "8.x"
```

Comment:
Please update the matrix names to include the Node version.  For example, `Linux (AzureCloud)` should be updated to something like `Linux Node8 (AzureCloud)` or similar.  This allows us to quickly determine which Node version was used in a job.